util.py

```diff
@@ -238,9 +238,13 @@ def get_default(field):
     if field == 'identity':
         return uuid.uuid4()
     elif field == 'timestamp_ms':
-        return -2e48 - 1
+        #this int64 field will be a float64 sometimes too (such as in the VBO
```

Comment:
I don't think this comment accurately explains why we chose this number.  It's just a big number that will have the same value whether represented as a floating point number or a signed int.  If we were going to document something, it might be "a magic number that represents positive/negative infinity."

Having said that, I don't think this should be documented here.  This is a client helper library that is just abiding by the rules of the API.  There will be API documentation that discusses timeless entities and how they are represented.  So rather than saying, "because the API works this way," I think we can just leave the comment out.